cmd/nfd-master/main.go

```diff
@@ -149,6 +147,8 @@ func initFlags(flagset *flag.FlagSet) (*master.Args, *master.ConfigOverrideArgs)
 			" DEPRECATED: will be removed in a future release along with the deprecated gRPC API.")
 	flagset.IntVar(&args.MetricsPort, "metrics", 8081,
 		"Port on which to expose metrics.")
+	flagset.IntVar(&args.GrpcHealthPort, "grpc-health", 8082,
+		"Port on which to expose the health endpoint.")
```

Comment:
nit:  grpc health endpoint.

Reply:
I am fine with calling it `grpc-health-endpoint`. It's just that `metrics` is also only `metrics` and not `metrics-endpoint` :)

Reply:
ah i meant in the flag info :)